app/models/language.rb

```diff
@@ -127,12 +125,11 @@ def self.score_lines(text)
   def self.update_recent_translations
     cutoff = @@last_update
     @@last_update = Time.zone.now
-    for locale, tag, text in Language.connection.select_rows(%(
-      SELECT locale, tag, text
-      FROM translation_strings t, languages l
-      WHERE t.language_id = l.id
-        AND t.updated_at >= #{Language.connection.quote(cutoff)}
-    ))
+    strings = TranslationString.joins(:language).
+              where(TranslationString[:updated_at].gteq(cutoff)).
+              pluck(Language[:locale], :tag, :text)
+
+    for locale, tag, text in strings
       TranslationString.translations(locale.to_sym)[tag.to_sym] = text
     end
```

Comment:
```suggestion
    strings.each do |locale, tag, text|
      TranslationString.translations(locale.to_sym)[tag.to_sym] = text
    end
```
Another supressed RuboCop offense.